lib/registries/addon/components/registries-license-picker/template.hbs

```diff
@@ -35,25 +35,24 @@
         </div>
         
         {{#if @manager.selectedLicense}}
-            {{#if @manager.requiredFields}}
-                <br>
-            {{/if}}
             <form.custom
+                local-class='AdditionalFields'
                 @changeset={{@manager.changeset}}
                 @valuePath='nodeLicense'
             >
                 {{#each @manager.requiredFields as |key|}}
                     <div class='form-group'>
-                        <label>
+                        <label for={{key}}>
```

Comment:
The label `for` attribute and Input `id` attribute should match, or the input element should be inside the label element. Just had this open in my browser for the accessibility audit https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

Reply:
Oh yup, you're absolutely right! Thanks for the catch!
Part of what I'd changed before was moving the input out of the label. Should it be moved back in? A side effect was that the input became full-sized once it was moved out (as opposed to the smaller sized textbox it usually is)

Reply:
either style is fine (`<label for='...'>` or input nested under label) but when using `for=` we shouldn't be building the same `id` twice in the template -- the `let` helper lets us build it once, and we have a [`unique-id` helper to guarantee uniqueness](https://github.com/aaxelb/ember-osf-web/blob/a6a725ac3612a49da405395edd5a6d416524ecfb/lib/osf-components/addon/helpers/unique-id.ts#L5-L18)